commands/build.go

```diff
@@ -165,3 +171,12 @@ func PullTemplates(templateURL string) error {
 	}
 	return err
 }
+
+func compileArg(argOpts []string) (map[string]string, error) {
+	arguments, err := parseMap(argOpts, "build-arg")
```

Comment:
Interesting re-use of this function. I added a new one in my version, but maybe you can send a PR to the master now and remove my newly added method that parses the values?

Reply:
@alexellis sure I'm on it :)